Resources/views/Collector/migrations.html.twig

```diff
@@ -0,0 +1,192 @@
+        </tr>
+        {% if collector.data.table is defined %}
+            <tr>
+                <td class="font-normal">Table Name</td>
+                <td class="font-normal">{{ collector.data.table }}</td>
+            </tr>
+        {% endif %}
+        {% if collector.data.column is defined %}
+            <tr>
+                <td class="font-normal">Column Name</td>
+                <td class="font-normal">{{ collector.data.column }}</td>
+            </tr>
+        {% endif %}
+    </table>
+    <table>
+        <thead>
+            <tr>
+                <th colspan="2" class="colored font-normal">Database</th>
+            </tr>
+        </thead>
+        <tr>
+            <td class="font-normal">Driver</td>
+            <td class="font-normal">{{ collector.data.driver }}</td>
+        </tr>
+        <tr>
+            <td class="font-normal">Name</td>
+            <td class="font-normal">{{ collector.data.name }}</td>
+        </tr>
+    </table>
+    <table>
+        <thead>
+            <tr>
+                <th colspan="2" class="colored font-normal">Migration Namespaces</th>
+            </tr>
+        </thead>
+        {% for namespace,directory in collector.data.namespaces %}
+            <tr>
+                <td class="font-normal">{{ namespace }}</td>
+                <td class="font-normal">{{ directory }}</td>
+            </tr>
+        {% endfor %}
+    </table>
+
+    <h3>Migrations</h3>
+    <table>
+        <thead>
+            <tr>
+                <th class="colored font-normal">Version</th>
+                <th class="colored font-normal">Description</th>
+                <th class="colored font-normal">Status</th>
+                <th class="colored font-normal">Executed at</th>
+                <th class="colored font-normal">Execution time</th>
+            </tr>
+        </thead>
+        {% for migration in collector.data.new_migrations %}
+            <tr>
+                <td class="font-normal">
+                    {% set version = migration.version|split('\\') %}
+                    {% set file = collector.data.namespaces[version|first] ~ '/' ~ version|last ~ '.php' %}
+                    <a href="{{ file|file_link(1) }}" title="{{ file }}">
+                        {{ migration.version }}
+                    </a>
+                </td>
+                <td class="font-normal">{{ migration.description }}</td>
+                <td class="font-normal"><span class="label status-error">NOT EXECUTED</span></td>
+                <td class="font-normal">n/a</td>
+                <td class="font-normal">n/a</td>
+            </tr>
+        {% endfor %}
+
+        {% set unavailable = [] %}
+        {% for migration in collector.data.executed_unavailable_migrations %}
+            {% set unavailable = unavailable|merge([migration.version]) %}
+        {% endfor %}
+
+        {% for migration in collector.data.executed_migrations|reverse %}
+            {% set is_unavailable = migration.version in unavailable %}
```

Comment:
isn't this just `is_unavailable = migration in collector.data.executed_unavailable_migrations`, avoiding to build the `unavailable` list ?